cmd/linux.go

```
@@ -7,10 +7,13 @@ import (
 	"os/exec"
 )
 
-func systemShell() *exec.Cmd {
+func systemShell(marker string) *exec.Cmd {
 	shell := os.Getenv("SHELL")
 	if shell == "" {
 		shell = "/bin/sh"
 	}
-	return exec.Command(shell, "-i")
+
+	command := exec.Command(shell, "-i")
```

golangci-lint (2.13.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)